docs: add code hyperlinks in rpc onboarding functions + update wordings (#354)
iron-fish/website: docs/onboarding/rpc/wallet.md

}
`} />

##  <GithubCodeLink link="wallet/create"/> wallet/create

> Creates a new account in the wallet with the given name and optionally setting it as the default account.

Reviewer: ```suggestion
Creates a new account in the wallet with the given name and optionally sets it as the default account.
```

}
`} />

## <GithubCodeLink link="wallet/exportAccount"/> wallet/exportAccount

Exports the keys to the default account, or the named account if specified. 

> If `viewOnly = true`, the returned spending key will be null, but the spending key may also be null if exporting a view-only account.

Reviewer: The wording was already like this, but I think it might be better as:

```suggestion
`spendingKey` will be null in the response if `viewOnly = true` or if exporting a view-only account.
```
